services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -597,6 +598,58 @@ class ApplicationServiceITest : DatabaseTest() {
             }
         }
 
+        @Test
+        fun `when application is in Allu should initialize access for new contacts`() {
+            val cableReport =
+                createCableReportApplicationData(
+                    areas = listOf(aleksanterinpatsas),
```

Comment:
The areas are irrelevant here. The factory-function creates an area by default.

Reply:
Fixed by removing areas.